src/gateway/kong-enterprise/index.md

```diff
@@ -1,6 +1,86 @@
+log trails are [Workspace](/gateway/{{page.kong_version}}/admin-api/workspaces/reference) and [RBAC](/gateway/{{page.kong_version}}/admin-api/rbac/reference)-aware,
+providing {{site.base_gateway}} operators a deep and wide look into changes happening within
+the cluster.
+
+## Keyring and data encryption
+
+{{site.base_gateway}} allows you to store sensitive data fields, such as consumer secrets, in an encrypted format within the database. This provides encryption-at-rest security controls in a {{site.base_gateway}} cluster.
+
+This functionality provides transparent, symmetric encryption of sensitive data fields at rest. Transparency refers to the fact that, when enabled, encryption/decryption of data is done on-the-fly by {{site.base_gateway}} immediately before writing/immediately after reading from the database. Responses generated by the Admin API containing sensitive fields continue to show data as plaintext, and runtime elements of {{site.base_gateway}} (such as plugins) that require access to sensitive fields do so transparently, without requiring additional configuration.
+
+See [Keyring and data encryption](/gateway/{{page.kong_version}}/kong-enterprise/db-encryption/) for more information. 
+
+## Roles-based access control (RBAC)
+
+You can configure {{site.base_gateway}}'s [RBAC](/gateway/{{page.kong_version}}/admin-api/rbac/reference/) feature using the
+[Admin API](/gateway/{{page.kong_version}}/admin-api/) or [Kong Manager](/gateway/{{page.kong_version}}/configure/auth/rbac). RBAC allows you to configure the roles and permissions for admin users to access resources in {{site.base_gateway}}.
+
+There are 4 basic entities involving RBAC.
+
+- **User**: The entity interacting with the system. Can be associated with
+  zero, one or more roles.
+- **Role**: Set of permissions. Has a name and can be associated with zero, one or more permissions.
+- **role_endpoint**: A set of enabled or disabled actions (`read`, `create`, `update`, `delete`) for `endpoint`.
+- **role_entity**: A set of enabled or disabled actions (`read`, `create`, `update`, `delete`) for `entity`.
+
+## Workspaces
+
+[Workspaces](/gateway/{{page.kong_version}}/admin-api/workspaces/examples/) provide a way to segment or group {{site.base_gateway}} entities. Entities in a workspace are isolated from those in other workspaces. 
+
+## Consumer groups
+
+You can use [consumer groups](/gateway/{{page.kong_version}}/admin-api/consumer-groups/reference/) to manage custom rate limiting configuration for subsets of consumers. With consumer groups, you can define any number of rate limiting tiers and
+apply them to subsets of consumers, instead of managing each consumer
+individually.
+
+For example, you could define three consumer groups:
+* A "gold tier" with 1000 requests per minute
+* A "silver tier" with 10 requests per second
+* A "bronze tier" with 6 requests per second
+
+## Event hooks
+
+{% include_cached /md/enterprise/event-hooks-intro.md %}
```

Comment:
Using the include gives us a very long intro that I don't think we need here. I think it would make sense to shorten to:

> Event hooks are outbound calls from Kong Gateway. With event hooks, the Kong Gateway can communicate with target services or resources, letting the target know that an event was triggered. When an event is triggered in Kong, it calls a URL with information about that event. Event hooks add a layer of configuration for subscribing to worker events using the admin interface. Worker events are integrated into Kong Gateway to communicate within the gateway context. For example, when an entity is created, the Kong Gateway fires an event with information about the entity. Parts of the Kong Gateway codebase can subscribe to these events, then process the events using callbacks.
> 
> In Kong Gateway, these callbacks can be defined using one of the following “handlers”:
>
> * webhook
> * webhook-custom
> * log
> * lambda
> 
> [Learn more about event hooks &rarr;](/gateway/{{page.kong_version}}/admin-api/event-hooks/reference